Microsoft.Toolkit.Uwp.SampleApp/SamplePages/Menu/Menu.bind

```diff
@@ -23,11 +25,14 @@
       <RowDefinition Height="auto" />
     </Grid.RowDefinitions>
 
-    <controls:Menu AllowTooltip="@[AllowTooltip:Bool:True]"
+    <controls:Menu
+                  AllowTooltip="@[AllowTooltip:Bool:False]"
```

Comment:
Could the formatting be fixed up in the bind file, it's a bit difficult to read?